ui/src/actions/literature.ts

```diff
@@ -343,14 +346,20 @@ export function curateReference({
         'query',
         'page',
       ]);
+
       // update references to see changes immediately
       const query = {
         ...{
           size,
           page,
         },
       };
-      dispatch(fetchLiteratureReferences(recordId, query))
+      dispatch(fetchLiteratureReferences(recordId, query)).then(() => {
+        // scroll to previous view
+        const element = getState().ui.get('scrollElement');
```

Comment:
Just to make sure we don't mismatch it in the future:
```suggestion
        const element = getState().ui.get('referenceListActiveElement');
```

WDYT?